public/index.html

```diff
@@ -49,8 +49,14 @@ <h4> Create a Geolocating Web Annotation </h4>
                 </a>
                 
             </div>
+	    <footer>
```

Comment:
The site already uses a footer called `geo-footer` so you should not use another footer element.